pystac_monty/sources/usgs.py

```diff
@@ -372,7 +373,7 @@ def make_source_event_item(self) -> Item:
         monty.hazard_codes = ["GH0004"]  # Earthquake surface rupture code
 
         # TODO Get country code from event data or geometry
-        country_codes = [self.geocoder.get_iso3_from_geometry(point)]
+        country_codes = ["UNK"]  # [self.geocoder.get_iso3_from_geometry(point)]
```

Comment:
We should store these constants somewhere and add notes on why we are using this. @ranjan-stha can you also add update gdacs empty country codes with this same value.

Reply:
Done.